### ISSUE TITLE
Adding Rule page

### ISSUE THREAD

**Issue by skystar-p** [sonowz/graduate-adventure#71]
## Rule Page

Currently, rule is just using pre-constructed YAML file.

But this preconstrucing rule file needs too much efforts, and is error-prone.

Furthermore, we cannot cover all of that rules, so we **should** implement Rule Page.

This issue references #5 .

Rule Page should contains:

- Select specific rule file and view by tree
- Expanding, collapsing tree interface
- Generating tree (this should be the most tough work)
- Tester, and vote (for preventing trolling)

Maybe this can be done in very end of this project.

- [ ] Make template of Rule Page
- [ ] Implementing Rule file selector and tree viewer
- [ ] Rule generator interface
- [ ] Voting system

Please request any of adding and deleting above list, and even opinion of rejecting this issue.

**Comment by sonowz** [comment]
Also, users should be able to choose what rule to apply. (If we use voting system)
This may be implemented in `/login` page, or in a new page.

**Comment by sonowz** [comment]
It could be better if we don't use user voting system and `admin` picks best working rule, because I don't think most users want to inspect rules carefully.

**Comment by skystar-p** [comment]
@dnsdhrj I agree about this: `because I don't think most users want to inspect rules carefully.`, but admin-based picking system will make our work harder.

Any ideas about this?
